frontend/test/features/home/home_page_test.dart

```diff
@@ -27,36 +29,36 @@ void main() {
 
     testWidgets('should show deposit button', (tester) async {
       await tester.pumpWidget(
-        WidgetHelpers.testableWidget(child: HomePage()),
+        WidgetHelpers.testableWidget(child: const HomePage()),
       );
 
-      expect(find.text('Deposit'), findsOneWidget);
+      expect(find.widgetWithText(FilledButton, 'Deposit'), findsOneWidget);
```

Comment:
ran into some test failures because button text was sometimes found elsewhere on the page.. went ahead and changed other occurrences of finding button text to be consistent